### DISCUSSION TITLE
Pinecil power

### DISCUSSION THREAD

**Discussion by Frenzer0** [Ralim/IronOS#906] [Q&A]
Hello, in the pinecil wiki it is written: 

> The first batch of Pinecil's were rated 12-24v @ 65W. After some heated discussion on the discussion group, it was decided that it would be downgraded to 12-21V @ 60W, due to concern over the connection of the DC jack to the USB-PD chip, which has a recommended maximum of 21v, and absolute maximum of 28v.

Can I use this charger knowing it's 65W? Or does it risk damaging the pinecil ?

https://fr.aliexpress.com/item/4000868717006.html?spm=a2g0o.productlist.0.0.c36542c4nYuVEl&algo_pvid=18844d1e-b804-4e83-ad6e-63f87fd26f13&algo_expid=18844d1e-b804-4e83-ad6e-63f87fd26f13-0&btsid=2100bddd16163487762417648e3706&ws_ab_test=searchweb0_0,searchweb201602_,searchweb201603_

Thanks

**Comment by Ralim** [comment]
Yep you should be fine with that.

You can always go higher on the wattage, it just won't use the extra wattage available.

**Comment by Frenzer0** [comment]
Thank you for these details, the charger being GaN Tech, I was not sure whether it damages the soldering iron or not, I do not know this technology

Is this only valid for the pinecil? I have always read that when an AC adapter exceeds the volts for a device it damages the device, but the Pinecil is 20V so it's good for the volts, but it exceeded the watt so I didn't know

**Reply by Ralim** [reply]
The rule for _all_ power adapters is:
* Voltage must match
* Current must meet or exceed
* Wattage must meet or exceed

A device will only pull as much current as it requires so having an adapter that can supply excess wont harm it but can mean a larger than required charger

**Comment by jagravhu** [comment]
The earlier, v2.13 firmware version is working well, with my Baseus  012QC type charger (from China), but the new, 2.14 version is not,  - the charger is not change the voltage up. The voltage remains at only 5V, and it is too low.
(The 2.13 version is switches up to 12V well.)

**Reply by Ralim** [reply]
Ah oh joy, another QC bug.
* Do you have any means to capture the analog voltages during the QC negotiation ?
* What hardware are you using (to confirm)
* Have you done a bisect to confirm what commit broke qc for you?

-> Looking at the changes there wasnt any large refactors to the QC itself, but the time it starts at may have changed as this was moved to its own thread.

**Comment by mark1352** [comment]
I ordered this power supply on amazon 
It is 74 watts and adjustable from 4 to 24 volts

https://www.amazon.com/dp/B08BL4QMGM/ref=cm_sw_em_r_mt_dp_J4K1RNFNBMNC3FK4T61J?_encoding=UTF8&psc=1
